x-pack/test/api_integration/apis/upgrade_assistant/es_deprecations.ts

```diff
@@ -60,45 +40,6 @@ export default function ({ getService }: FtrProviderContext) {
           }
         });
       });
-
-      describe('index setting deprecation', () => {
```

Comment:
This test was originally added to guard against https://github.com/elastic/kibana/pull/123599. Since we no longer rely on the message string from ES, this test is no longer necessary,